helpers/press-release-sparql-queries.js

```diff
@@ -0,0 +1,140 @@
+import { sparqlEscapeString, sparqlEscapeUri, sparqlEscapeDateTime, uuid } from 'mu';
+import { querySudo as query } from '@lblod/mu-auth-sudo';
+
+const PREFIXES = `
+		PREFIX mu: ${sparqlEscapeUri('http://mu.semte.ch/vocabularies/core/')}
+		PREFIX ebucore: ${sparqlEscapeUri('http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#')}
+		PREFIX ext: ${sparqlEscapeUri('http://mu.semte.ch/vocabularies/ext/')}
+		PREFIX vlpt: ${sparqlEscapeUri('http://themis.vlaanderen.be/id/publicatie-taak/')}
+		PREFIX adms: ${sparqlEscapeUri('http://www.w3.org/ns/adms#')}
+		PREFIX dct: ${sparqlEscapeUri('http://purl.org/dc/terms/')}
+		PREFIX prov: ${sparqlEscapeUri('http://www.w3.org/ns/prov#')}
+		`;
+
+export async function findPressReleasesWithPublicationEvents(id) {
```

Comment:
Aangezien dit maar 1 resultaat teruggeeft, is `findPressReleaseWithPublicationEvent` waarschijnlijk een betere naam